lib/ansible/modules/remote_management/ansible_playbook.py

```diff
@@ -0,0 +1,469 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Fabien Dupont <fdupont@redhat.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+DOCUMENTATION = '''
+---
+module: ansible_playbook
+short_description: Launch Ansible playbook on remote host
+description:
+  - This module launches an Ansible playbook. It is mainly a
+    wrapper for the ansible-playbook command, so the options
+    match the ansible-playbook options.
+author:
+  - Fabien Dupont (@fdupont-redhat)
+version_added: "2.7"
+options:
+  path:
+    description:
+      - Path of the playbook to run
+    required: true
+    aliases: [name]
+  inventory_hosts:
+    description:
+      - Specify inventory as a comma separated host list
+      - Either `inventory_hosts` or `inventory_path` is required.
+  inventory_path:
+    description:
+      - Specify inventory path (file or directory)
+      - Either `inventory_hosts` or `inventory_path` is required.
+  limit:
+    description:
+      - Further limit selected hosts to an additional pattern
+    required: false
+  extra_vars:
+    description:
+      - Set additional variables as key=value
+    required: false
+  extra_vars_files:
+    description:
+      - Set additional variables from files
+    required: false
+  tags:
+    description:
+      - Only run plays and tasks tagged with these values
+    required: false
+  skip_tags:
+    description:
+      - Do not run plays and tasks tagged with these values
+    required: false
+  connection_method:
+    description:
+      - Connection type to use
+    choices:
```

Comment:
Instead of listing here, point at `ansible-doc -t conneciton -l` and maybe link to the connection plugins page, since this can change by installation.

Reply:
That would be the case for all options as it is a wrapper for ansible-playbook, wouldn't it ?

Reply:
yep

Reply:
@bcoca 
Thinking about it, the problem with linking to the documentation without providing the actual list is that the CI will complain that implementation and documentation are not consistent. Should I just add the link to the documentation and keep the list ?